modules/bullet/space_bullet.cpp

```diff
@@ -28,6 +28,9 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+//just for debug
+#include <iostream>
```

Comment:
You can use `print_line` for debug. And debug code should be removed eventually before merge.